src/pages/en/guides/server-side-rendering.md

```diff
@@ -132,3 +139,75 @@ export async function get({ params }) {
   });
 }
 ```
+
+While in this example, a valid HTML code can be returned to render the page or it's content, API Routes can be used as REST API Endpoints.
+
+### Serverless Functions
+
+API Routes can be used to run [serverless functions](https://en.wikipedia.org/wiki/Serverless_computing#Serverless_runtimes) without exposing sensitive data to the clients, such as authentications, database access, verifications, etc.
```

Comment:
```suggestion
API outes can be used to run [serverless functions](https://en.wikipedia.org/wiki/Serverless_computing#Serverless_runtimes) (e.g. authentications, database access, and verifications) without exposing sensitive data to the client.
```
(Feels like client should be singular here? And, I think the list of examples reads better coming directly after the thing they are examples of.)